docs/sources/getting-started/glossary.md

```diff
@@ -40,12 +40,36 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Export/Import</td>
```

Comment:
```suggestion
    <td style="vertical-align: top">Export/Import dashboard</td>
```

While I can't think of another resource that we "export"  or "import", it's a common terminology and is likely used or will be used for other things as well. Would it make sense to make it a little more specific to exporting/importing _dashboards_.